### PR TITLE
LOOP-921 Dash sdk update

### PR DESCRIPTION
**Pull request by ps2**
https://tidepool.atlassian.net/browse/LOOP-921

### CODE REVIEW

File: DashKit.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -912,6 +953,24 @@
 			};
 			name = Release;
 		};
+		C11615F1239588DB0039A072 /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				CODE_SIGN_STYLE = Automatic;
+				DEVELOPMENT_TEAM = UY678SP37Q;
```
Comment (darinkrauss) [inline DashKit.xcodeproj/project.pbxproj:960]: I believe the `DEVELOPMENT_TEAM` should be `""`.

File: DashKit.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -912,6 +953,24 @@
...
+				PRODUCT_NAME = "$(TARGET_NAME)";
+			};
+			name = Debug;
+		};
+		C11615F2239588DB0039A072 /* Release */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				CODE_SIGN_STYLE = Automatic;
+				DEVELOPMENT_TEAM = UY678SP37Q;
```
Comment (darinkrauss) [inline DashKit.xcodeproj/project.pbxproj:969]: I believe the `DEVELOPMENT_TEAM` should be `""`.

**Review by darinkrauss — CHANGES_REQUESTED** [review]
The `DEVELOPMENT_TEAM` was set when it should be blank. Also, did you intend on including the latest changes to the `dash-sdk-ios` repo?

File: DashKit.xcodeproj/project.pbxproj (hunk 1 above)
Comment (ps2) [reply DashKit.xcodeproj/project.pbxproj:960]: I don't think we need to have special rules for this, as we do with repos intended to be shared with the DIY community. Would prefer to keep it this way to streamline running tests.

Comment (ps2) [reply DashKit.xcodeproj/project.pbxproj:960]: And yes, this branch is the dash-sdk-update branch; added the Cartfile target as a tagalong.

**Review by darinkrauss — APPROVED** [review]
LGTM! 🚢

**Pull request merged**